libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -93,7 +93,7 @@ export class CosmosAPI {
       url: `${this.defaultEndpoint}/cosmos/base/tendermint/${this.version}/blocks/latest`,
     });
 
-    return data.block.header.height;
+    return parseInt(data.block.header.height);
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt:
> If 0 or not provided, the radix will be inferred based on string's value. Be careful — this does not always default to 10!
```suggestion
    return parseInt(data.block.header.height, 10);
```

Reply:
Or you can use `Number()`
```suggestion
    return Number(data.block.header.height);
```

Reply:
You're right but the doc explains in more details:

> The radix argument is [converted to a number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_coercion). If it's unprovided, or if the value becomes 0, NaN or Infinity (undefined is coerced to NaN), JavaScript assumes the following:
> 
> - If the input string, with leading whitespace and possible +/- signs removed, begins with 0x or 0X (a zero, followed by lowercase or uppercase X), radix is assumed to be 16 and the rest of the string is parsed as a hexadecimal number.
> - If the input string begins with any other value, the radix is 10 (decimal).

Also in the function's doc in the code itself:

> A value between 2 and 36 that specifies the base of the number in string. If this argument is not supplied, strings with a prefix of '0x' are considered hexadecimal. All other strings are considered decimal.

`blockHeight` can't be expected to be hexadecimal, this should be safe.